lib/fluent/plugin/out_memcached.rb

```diff
@@ -33,12 +31,21 @@ def start
 
   def shutdown
     @memcached.close
+    super
   end
 
   def format(tag, time, record)
     [tag, time, record].to_msgpack
```

Comment:
In v0.14, we can include tag information and reduce chunk size.
Should this plugin reduce chunk size? 
@innossh How do you think of this question?